main/Main.hs

```diff
@@ -171,13 +176,16 @@ main = do
         . setServerName (toS $ "postgrest/" <> prettyVersion) $
         defaultSettings
 
+  whenLeft socketFileMode $
+    panic $ (fromJust . leftToMaybe) socketFileMode
+
   -- Checks that the provided proxy uri is formated correctly
   when (isMalformedProxyUri $ toS <$> proxy) $
     panic
       "Malformed proxy uri, a correct example: https://example.com:8443/basePath"
 
   -- Checks that the provided jspath is valid
-  when (isLeft roleClaimKey) $
+  whenLeft roleClaimKey $
```

Comment:
I think you changed an unrelated line here.

Reply:
I refactored based on the `whenLeft` suggestion you made, to keep it consistent